debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -496,7 +497,7 @@ public void canInfluenceTableColumnTypes() {
         assertThat(valueSchema.field("payload").schema().type()).isEqualTo(SchemaBuilder.bytes().type());
         assertThat(valueSchema.field("bool").schema().type()).isEqualTo(SchemaBuilder.bool().type());
 
-        assertThat(((Struct) eventRouted.value()).get("payload")).isEqualTo("{}".getBytes());
+        assertThat(asStruct(eventRouted.value()).get("payload")).isEqualTo("{}".getBytes());
```

Comment:
I know that this change could be considered as "refactoring" which should be discussed first, so I am happy to revert it if needed

Reply:
Hi @pmalon, would it be possible to at least separate the `asStruct` method change into a separate commit?  That way we can easily backport the real bugfix and keep the refactoring commit in the main branch?